outfit.js

```diff
@@ -2,7 +2,7 @@ class Outfit {
   constructor(id){
     this.id = id;
     this.title = '';
-    this.garments = [null, null, null, null];
+    this.garments = [null, null, null];
```

Comment:
Good call, we also don't need that extra space anymore since the backgrounds have their own property.